graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -184,6 +185,11 @@ private DirectIndexQueryResult performAsyncDirectIndexQuery(SearchParams searchP
                 // then we need to delete the previous search context async
                     processRequestWithSameSearchContextId(searchParams);
             }
+
+            String KeepAliveTime = AtlasConfiguration.INDEXSEARCH_ASYNC_SEARCH_KEEP_ALIVE_TIME_IN_SECONDS.getLong() +"s";
```

Comment:
If we are adding it here remove the same existing logic from func `submitAsyncSearch`

Reply:
done